idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/coercions/Utils.kt

```diff
@@ -27,7 +28,7 @@ internal fun KtDotQualifiedExpression.implicitParticipatingTypes(): Pair<KotlinT
 internal fun CompilerContext.coercionProofMessage(ktDotQualifiedExpression: KtDotQualifiedExpression): String =
   ktDotQualifiedExpression.implicitParticipatingTypes()?.let { (subtype, supertype) ->
     coerceProof(subtype, supertype)?.coercionMessage()
-  } ?: "Proof not found"
+  } ?: MetaIdeBundle.message("proofs.markers.proof.not.found")
```

Comment:
Out of scope of this PR, but did you discover why sometimes you're working with the Proofs and then it randomly starts saying "Proof not found" at some point? Looks like if our algorithm to read the proofs or the mechanism we use to store them would be mutated over time or something.